A89-xds-cluster-metric-label.md

```diff
@@ -61,6 +61,8 @@ variable protection is unnecessary.
 gRFC A78 added `grpc.lb.locality` to per-call and WRR metrics, while this is
 only adding the new label to per-call. Cluster is an xDS-specific concept, so
 it is more awkward to add to WRR and that is left as potential future work.
+FIXME: the name was changed so is no longer an xds-specific concept and there is
```

Comment:
I'd be fine also adding this to the WRR metrics while we're at it.

Probably not worth adding it to the PF metrics, since @yashykt will presumably be working on a gRFC to replace the PF metrics with the new subchannel metrics that we discussed today.

Reply:
Yeah, I saw that locality was only WRR, and then I was wondering why locality wasn't part of pick-first metrics and vice-versa for this for WRR. I'll work with Yash to get the dust to settle, but overall I'm not worried.

Reply:
There wasn't any real reason not to add the locality label to PF metrics; it was just that no one had asked for it.